src/main/java/mate/jdbc/service/CarServiceImpl.java

```diff
@@ -0,0 +1,76 @@
+package mate.jdbc.service;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.CarDao;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.util.ConnectionUtil;
+
+@Service
+public class CarServiceImpl implements CarService {
+    @Inject
+    private CarDao carDao;
+
+    @Override
+    public Car create(Car car) {
+        return carDao.create(car);
+    }
+
+    @Override
+    public Optional<Car> get(Long id) {
+        return carDao.get(id);
+    }
+
+    @Override
+    public List<Car> getAll() {
+
+        try {
+            return carDao.getAll();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        return null;
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        return carDao.delete(id);
+    }
+
+    @Override
+    public Car update(Car car) {
+        return carDao.update(car);
+    }
+
+    @Override
+    public List<Car> getAllByDriver(Long driverId) {
+        return carDao.getAllByDriver(driverId);
+    }
+
+    public void addDriverToCar(Driver driver, Car car) {
+        car.getDrivers().add(driver);
+        carDao.update(car);
+    }
+
+    @Override
+    public void removeDriverFromCar(Driver driver, Car car) {
+        String query = "DELETE from cars_drivers "
+                + "WHERE car_id = ? AND driver_id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, car.getId());
+            statement.setLong(2, driver.getId());
+            statement.executeUpdate();
+        } catch (SQLException e) {
+            throw new DataProcessingException(
+                    "Couldn't remove drivers from car by car id  " + car.getId(), e);
+        }
```

Comment:
? 
are you sure it should be in service?
```suggestion
        car.getDrivers().remove(driver);
        carDao.update(car);
```